.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"    
+  },
+  "extends": ["airbnb-base",
+    "eslint:recommended",
+    "plugin:react/recommended"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off"
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ ] Use [this one](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) instead.

Reply:
Hi, I used that one before but got errors, "ecmaVersion is deprecated" or something like that. There is no a certain solution for it in internet, so that's why I used this one instead. The other one is not working.